lib/broccoli/rollup-with-dependencies.ts

```diff
@@ -35,7 +38,17 @@ class RollupWithDependencies extends Rollup {
           ]
         ],
         plugins: [
-          'external-helpers'
+          'external-helpers',
+          [DebugMacros, {
+            envFlags: {
+              source: '@glimmer/env',
+              flags: { DEBUG: !isProduction, PROD: isProduction, CI: !!process.env.CI }
```

Comment:
@mmun - We recently had to add a CI flag in glimmer-vm to suppress the damned debugSlice noise in CI because it was emitting too much logging and causing our CI builds to fail when they hit the 4MB log size limitation in Travis.